common/logisticspipes/gui/orderer/GuiRequestTable.java

```diff
@@ -640,6 +642,15 @@ public void handleSimulateAnswer(Collection<IResource> used, Collection<IResourc
 
 	@Override
 	protected void keyTyped(char c, int i) throws IOException {
+		if (search.isFocused()){
+			if (!search.isEmpty() && search.handleKey(c, i))
+				return;
+		} else if (GuiScreen.isAltKeyDown() && ChatAllowedCharacters.isAllowedCharacter(c)) {
+			this.mouseClicked(search.getLeft(), search.getTop(), 0); // itemDisplay.requestCountBar.setFocus(false); & search.setFocused(true);
```

Comment:
What's that comment for?

Reply:
TODO: Find a more correct method, instead of mouseclick
itemDisplay.requestCountBar is protected